src/components/Pagination.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+
+class Pagination extends Component{
+  
+  render(){
+    return(
+      <div>
+        <ul className="pagination">
+          <li className="page-item disabled">
+            <a className="page-link" href="#" >&laquo;</a>
+          </li>
+          <li className="page-item active">
+            <a className="page-link" href="#" onClick = {this.props.handleClick}>1</a>
+          </li>
+          <li className="page-item">
+            <a className="page-link" href="#" id='2' onClick = {this.props.handleClick}>2</a>
```

Comment:
We can store the data in `id` or assign it right to the method.

As these are semantically more buttons than anchors, what about changing them that way. (This will prevent warnings about `href="#"`)

In components accepting props, please define proptypes https://reactjs.org/docs/typechecking-with-proptypes.html

Code style comments: 
- please do not add spaces when assigning props/attributes like this `onClick = this.something`
- check if your editor has configuration to automatically add empty line at the end of the file, as thats the standard practice (so you do not have these red signs in the last line of the file in github changelog)
- take a look at your indentation/empty lines between parts of the code, as there are some unnecessary ones.

`<button className="page-link" onClick={() => this.props.handleClick(2)}>2</a>`

Reply:
I set the automatic adding empty line

Reply:
For pagination to work in the right way we need add more advance logic to cover all functionallty. Should I focus on this or not for now?